clusterloader2/testing/load/config2.yaml

```diff
@@ -0,0 +1,276 @@
+# TODO(https://github.com/kubernetes/perf-tests/issues/413): Rename to config.yaml and delete the old one once tested.
```

Comment:
nit: can we have this file renamed more descriptively? It's hard to tell difference between config and config2. What about config_with_svc_bundle or sth similar?

Reply:
experimental_config maybe?

[I'm not sure people should understand it up until now]

Reply:
Changed to experimental_config.